src/iiif/views.py

```diff
@@ -51,6 +52,7 @@ def send_dataportaal_login_url_to_burger_email_address(request):
         tools.send_email(payload['email'], email_subject, email_body)
 
     except tools.ImmediateHttpResponse as e:
+        log.error(str(e.response.content))
```

Comment:
Same: `log.exception("Some useful description")` is hier speciaal voor gemaakt. Die pakt de stacktrace info zelf.